.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: CI Pipeline
+on: [push, pull_request]
+
+jobs:
+  unit-tests:
+    name: Unit tests
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```

Comment:
> macOS-latest workflows will use macOS-11 soon. For more details, see https://github.com/actions/virtual-environments/issues/4060

@atilaneves @John-Colvin shall I replace `macOS-latest` with `macOS-10.15`? Given https://issues.dlang.org/show_bug.cgi?id=21919 I think only dmd-nightly would work on macOS 11 (as of https://github.com/dlang/druntime/pull/3612).